idp-service/src/test/java/io/mosip/idp/AuthCodeFlowTest.java

```diff
@@ -203,11 +220,13 @@ private OAuthDetailResponse getOauthDetails(String clientId, String redirectionU
         OAuthDetailRequest oAuthDetailRequest = new OAuthDetailRequest();
         oAuthDetailRequest.setClientId(clientId);
         oAuthDetailRequest.setRedirectUri(redirectionUrl);
-        oAuthDetailRequest.setAcrValues("level0 level1");
+        oAuthDetailRequest.setAcrValues("level0 mosip:idp:acr:static-code");
```

Comment:
there is no level 0 correct ?

Reply:
Its test case Vishwanath, invalid/unknown ACR are ignored.

Reply:
But it will confuse the developer when they are following the code, can we rather use a valid value ?